.github/workflows/verify.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           cache: maven
       - name: Install
         id: install
-        run: $MVN install
+        run: $MVN install -Pexaples
```

Comment:
There's a typo here: Should be `-Pexamples`

Reply:
Nice catch. Should be fixed now. My "N" and "M" key are on the fritz. I should be getting a new laptop soon though :)